docs/source/chapters/about.rst

```diff
@@ -47,22 +46,20 @@ Some of the pgRouting library core features are:
 * `Johnson's Algorithm <https://en.wikipedia.org/wiki/Johnson's_algorithm>`__
 * `Floyd-Warshall Algorithm
   <https://en.wikipedia.org/wiki/Floyd%E2%80%93Warshall_algorithm>`__
-* `A* Algorithm <https://en.wikipedia.org/wiki/A*_search_algorithm>`__
-* `Bi-directional Algorithms
-  <https://en.wikipedia.org/wiki/Bidirectional_search>`__
-  * Bi-directional Dijkstra
-  * Bi-directional A*
-* `Traveling Sales Person
+* `A* Search Algorithm <https://en.wikipedia.org/wiki/A*_search_algorithm>`__
+* `Bi-directional Search Algorithms <https://en.wikipedia.org/wiki/Bidirectional_search>`__
+   * Bi-directional Dijkstra
+   * Bi-directional A*
+* `Traveling Salesperson Problem
   <https://en.wikipedia.org/wiki/Travelling_salesman_problem>`__
 * Driving Distance
 * many more!!!
 
-pgRouting is Open Source, available under the GPLv2 license and is supported and
+pgRouting is an Open-Source Software, available under the GPLv2 license and is supported and
 maintained by `Georepublic <http://georepublic.info>`__, `iMaptools
 <http://imaptools.com/>`_ and a broad user community.
```

Comment:
Reginas company is missing

Reply:
Open source is 2 words

Reply:
Please provide their url too. Will be updated.

Reply:
Is it the correct name and link?
Name - Paragon Corporation
Link - https://www.paragoncorporation.com/

Reply:
yes it is